GoogleUtilities/README.md

```diff
@@ -1,39 +0,0 @@
-# Google Utilities
-
-## Overview
-
-The GoogleUtilities pod is a set of utilities organized into CocoaPods subspecs.
-See the [podspec](../GoogleUtilities.podspec) for a summary of what utilities
-are currently included. They're used by Firebase and other Google products.
-
-Direct usage by non-Google products and CocoaPods is **NOT** currently
-recommended or supported in general.
-
-**However**, we do specifically recommend that non-Google SDKs can use the App Delegate Swizzler to hook into app delegate methods as this reduces conflicts with multiple SDKs trying to do the same.
```

Comment:
@maksymmalyhin What do you think of this recommendation? Do we want to encourage this?

Reply:
Probably not once we have an alternative ready. Thank you for the reminder to prioritize work on [MulticastAppDelegate](https://github.com/google/GoogleUtilities/blob/mm/app-delegate-proxy/GoogleUtilities/MulticastAppDelegate/MulticastAppDelegate.swift) for our next iterations!